src/common.jl

```diff
@@ -48,6 +48,24 @@ end
     WS #Wind Turbine, Offshore
 end
 
+"AER Aggregated Fuel Code From https://www.eia.gov/survey/form/eia_923/instructions.pdf"
+@enum ThermalFuels begin
+    COL	#Anthracite Coal and Bituminous Coal
+    WOC	#Waste/Other Coal (includes anthracite culm, gob, fine coal, lignite waste, waste coal)
+    DFO	#Distillate Fuel Oil (Diesel, No. 1, No. 2, and No. 4
+    WOO	#Waste Oil Kerosene and JetFuel Butane, Propane,
+    PC  #Petroleum Coke
+    RFO	#Residual Fuel Oil (No. 5, No. 6 Fuel Oils, and Bunker Oil)
+    NG	#Natural Gas
```

Comment:
Can we use full words?  NATURAL_GAS

Reply:
These are the codes that are published by the Energy Information Agency and in many cases, the data will be in this format. I hate the codes because they are somewhat cryptic, but I'm not entirely opposed since they are standardized. Would full words get too verbose? `RESIDUAL_FUEL_OIL`...

Reply:
Clayton is right, these are the codes used in the data originally.

Reply:
That doesn't preclude defining a mapping of cryptic code to PowerSystems enum in the parsing code.  I suggest that it's better to be able to understand the main PowerSystems code without having to look up the comments.

Reply:
I added a different naming scheme that is more detailed.

Reply:
Looks good.  You might as well define the map of code-to-enum now.